tests/test_diffraction_DiffractionPattern.py

```diff
@@ -160,11 +160,79 @@ def test_k_values_and_k_vectors(self):
         dp = freud.diffraction.DiffractionPattern()
 
         for size in [2, 5, 10]:
-            for npoints in [10, 20, 75]:
-                box, positions = freud.data.make_random_system(npoints, size)
-                dp.compute((box, positions))
-
-                output_size = dp.output_size
-                npt.assert_allclose(dp.k_values[output_size // 2], 0)
-                center_index = (output_size // 2, output_size // 2)
-                npt.assert_allclose(dp.k_vectors[center_index], [0, 0, 0])
+            box, positions = freud.data.make_random_system(size, 1)
+            zoom = 4
+            view_orientation = np.asarray([1, 0, 0, 0])
+            dp.compute((box, positions), view_orientation=view_orientation, zoom=zoom)
+
+            output_size = dp.output_size
+            npt.assert_allclose(dp.k_values[output_size // 2], 0)
+            center_index = (output_size // 2, output_size // 2)
+            npt.assert_allclose(dp.k_vectors[center_index], [0, 0, 0])
+
+            # Tests for the left and right k_value bins. Uses the math for
+            # np.fft.n
+            # fftfreq and the _k_scale_factor to write an expression for
+            # the first and last bins' k-values and k-vectors.
+
+            scale_factor = 2 * np.pi * output_size / (np.max(box.to_matrix()) * zoom)
+
+            if output_size % 2 == 0:
+
+                first_k_bin = -0.5 * scale_factor
+                last_k_bin = (0.5 - (1 / output_size)) * scale_factor
```

Comment:
Maybe these variable names would be better?
```suggestion
                first_k_value = -0.5 * scale_factor
                last_k_value = (0.5 - (1 / output_size)) * scale_factor
```